WordPress/src/main/java/org/wordpress/android/ui/comments/CommentsActivity.java

```diff
@@ -91,12 +101,21 @@ public void onCreate(Bundle savedInstanceState) {
         } else {
             getIntent().putExtra(KEY_AUTO_REFRESHED, savedInstanceState.getBoolean(KEY_AUTO_REFRESHED));
             getIntent().putExtra(KEY_EMPTY_VIEW_MESSAGE, savedInstanceState.getString(KEY_EMPTY_VIEW_MESSAGE));
-
-            mSelectedCommentId = savedInstanceState.getLong(KEY_SELECTED_COMMENT_ID);
+            mComment = (CommentModel) savedInstanceState.getSerializable(KEY_SELECTED_COMMENT);
         }
     }
 
     @Override
+    public void onStart() {
+        super.onStart();
```

Comment:
TBH, I should spend a bit more time on this and drop `CommentActions.ChangeType changeType` completly, but that's a rabit hole, it will bring me to drop `mNote` completly from CommentsActivity/Fragment/List.